doc/tools/coqrst/notations/fontsupport.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
Have you checked that this code actually runs with python3?  IIRC, this file does not run with python3 and is not actually run at all, see the comment "This script is here for reference. It was used to generate the modified
font CoqNotations.ttf from UbuntuMono-B.ttf.
"

Reply:
I have not, but then based on your comment we should not keep untested code in the repos.

Reply:
We have no idea whether it runs or not. See comment below:

>This script is here for reference. It was used to generate the modified font CoqNotations.ttf from UbuntuMono-B.ttf.

Reply:
This code does not run with python3, as fontforge does not provide python3 bindings, only python2 bindings.  (The code requires adding `# -*- coding: utf-8 -*-` to run with python2, but otherwise it seems to run fine under python2.  Note that this coding line was originally present but was removed by @Zimmi48 in 1a9c769ed363ee2f2784e7252af72e6c1e2fbcc6 which caused this file to no longer work.)

I don't have a strong opinion about removing this file vs keeping it for documentation purposes.  (`git blame` claims this file was introduced by @maximedenes and edited by @cpitclaudel ; they might have opinions?)  However, changing `python2` to `python3` is incorrect, and should not be commited.

```suggestion
#!/usr/bin/env python2
# -*- coding: utf-8 -*-
```